### DISCUSSION TITLE
Example "todo" component demo fails when JavaScript is disabled

### DISCUSSION THREAD

**Discussion by petermjones** [django-commons/django-unicorn#636] [General]
Your example "todo" component demo fails when JavaScript is disabled.  You've got one demo on the main page, shouldn't it be production ready?  Why are the buttons outside the form tag?  And then, of course, you'll needs some CSRF support.

**Comment by adamghill** [comment]
Feel free to make a PR for the docs or for `Unicorn` itself for any fixes you'd like to see.
